### PR TITLE
Add plugin merge notes to UPGRADING.rst

### PR DESCRIPTION
**Pull request by bernd**
Some information about the big plugin merge.

### CODE REVIEW

File: UPGRADING.rst [hunk 1]
```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
...
+
+- `Beats Input <https://github.com/Graylog2/graylog-plugin-beats>`_
+- `CEF Input <https://github.com/Graylog2/graylog-plugin-cef>`_
+- `Collector Plugin <https://github.com/Graylog2/graylog-plugin-collector>`_
+- `Enterprise Integration Page <https://github.com/Graylog2/graylog-plugin-enterprise-integration>`_
+- `Map Widget <https://github.com/Graylog2/graylog-plugin-map-widget>`_
+- `NetFlow Input <https://github.com/Graylog2/graylog-plugin-netflow>`_
+- `Pipeline Processor <https://github.com/Graylog2/graylog-plugin-pipeline-processor>`_
+
+That means these plugins are not available as separate plugins anymore. If you manually update your Graylog installation (without using operating system packages), make sure to remove all old plugin files from the ``plugin/`` folder.
```
Comment (joschi) [inline UPGRADING.rst]: Let's add a reference to the `plugin_dir` configuration setting as the plugins may not reside in a directory named `plugin`.

File: UPGRADING.rst [hunk 2]
```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
 +---------------------------------+----------------------------------+--------------------+
 | ``web_tls_key_password``        | ``http_tls_key_password``        | Empty              |
 +---------------------------------+----------------------------------+--------------------+
+
+Plugins moved into the Graylog server
```
Comment (joschi) [inline UPGRADING.rst]: s/moved/merged/

(Move indicates that they're still plugins, but in a different place)

File: UPGRADING.rst [hunk 3]
```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
...
+=====================================
+
+In 3.0 we merged the following plugins into the server Graylog server repository:
```
Comment (joschi) [inline UPGRADING.rst]: Duplicate "server".

Alternative wording:
"Starting with Graylog 3.0.0, the following official plugins were merged into Graylog core:"

File: UPGRADING.rst [hunk 4]
```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
...
+- `CEF Input <https://github.com/Graylog2/graylog-plugin-cef>`_
+- `Collector Plugin <https://github.com/Graylog2/graylog-plugin-collector>`_
+- `Enterprise Integration Page <https://github.com/Graylog2/graylog-plugin-enterprise-integration>`_
+- `Map Widget <https://github.com/Graylog2/graylog-plugin-map-widget>`_
+- `NetFlow Input <https://github.com/Graylog2/graylog-plugin-netflow>`_
+- `Pipeline Processor <https://github.com/Graylog2/graylog-plugin-pipeline-processor>`_
+
+That means these plugins are not available as separate plugins anymore. If you manually update your Graylog installation (without using operating system packages), make sure to remove all old plugin files from the ``plugin/`` folder.
+
+The old issues in these repositories are still available for reference but new issues should only be created in the `Graylog server issue tracker <https://github.com/Graylog2/graylog2-server/issues`.
```
Comment (joschi) [inline UPGRADING.rst]: Broken link (missing `>`).

**Comment by bernd** [comment]
@joschi Thanks! :+1:  I fixed the issues.

**Pull request merged**